Sources/MapboxNavigation/GenericRouteShield.swift

```diff
@@ -16,6 +16,14 @@ public class GenericRouteShield: StylableView {
         }
     }
     
+    @objc public dynamic var highlightColor: UIColor? {
```

Comment:
does this property need to be added to the hash if `isHighlighted` is already included?

Reply:
I don't think so. Because the `isHighlighted` is include, and the `foregroundColor` and `borderColor` already included.